back/node_watcher/datamodel.prisma

```diff
@@ -172,4 +172,21 @@ type ReferendumStatus {
   blockNumber: BlockNumber! @relation(name: "ReferendumStatusAtBlockNumber", link: INLINE)
   referendum: Referendum! @relation(name: "ReferendumStatusFromReferendum", link: INLINE)
   status: String!
-}
\ No newline at end of file
+}
+
+type Treasury {
+  id: ID! @id
```

Comment:
You must have an `Int` here otherwise hasura will throw. I'd advise to carefully follow the same path as what I've done for proposal and referendum.

Reply:
done